docs/sphinx/reference-libobs-util-platform.rst

```diff
@@ -440,6 +440,12 @@ Other Functions
 
 ---------------------
 
+.. function:: uint64_t os_get_sys_total_size(void)
+
+   Returns the amount of memory installed.
```

Comment:
Should probably say unit. Also @notr1ch correctly remarked that this function name - while consistent with the rest - is terrible and we should probably change both since the other one isn't actually used by anything.